src/index.js

```diff
@@ -0,0 +1,36 @@
+const express = require('express');
+
+const axios = require('axios');
+
+// const clientID = process.env.CLIENT_ID;
+// const clientSecret = process.env.CLIENT_SECRET;
+
+const clientID = 'c820624d21ec78915700';
```

Comment:
We need to remove this clientID and somehow abstract it to an environment variable; same with `clientSecret`

Best way would probably be to place it in the `.env` file, and call it from there. 

When it's deployed, we'll be able to add Environment variables securely from the Vercel dashboard (or whatever service we end up using). 

I think it might be better to fix this before merging into `main`